modules/examples/todolist-http-http4s/src/main/scala/todo/peristence/Persistence.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package exapmles.todolist
+package peristence
+
+import freestyle.tagless.module
+import examples.todolist.persistence.{
```

Comment:
When we have more than 2 o 3 imports in the same package, we use to import all the package.

`import examples.todolist.persistence._`